README.md

```diff
@@ -767,8 +830,59 @@ You probably didn't update the font files linked to your native project after up
 
 Sometimes vendors decides to remove some icons from newer releases, this has nothing to do with this package. If you depend on an older version of a font you can add it as a [custom font](#custom-fonts).
 
+#### Web-pack complains about unsupport JSX Syntax
+
+You will need to add JSX support for react-native-vector-icons to your transpiler configuration e.g. babel. the list of modules that support JSX (if using webpack)
+
+This may look something like the following:
+
+```js
+// Process application JS with Babel.
+// The preset includes JSX, Flow, TypeScript, and some ESnext features.
+{
+  test: /\.(js|mjs|jsx|ts|tsx)$/,
+  include: [
+    paths.appSrc,
+    // START - support for JSX in react-native-vector-icons
```

Comment:
@doublethefish This is greatm thanks for contributing.

Could you add a note before the code indicating that only the bits between START and END must be added and that everything else depends on your particular project?

Just want to make it blazingly obvious that you shouldn'[t change all of your current config.

Reply:
No problems.

I've used a `diff` format instead and removed some of the surrounding context, which makes it much clearer. This follows a pattern used elsewhere in the README.

There were also a few other slightly cludgy bits of text that I've tidied.